src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,242 @@
+import classNames from 'classnames';
```

Comment:
@orlandohohmeier I've renamed the components so that they make more sense.
`Breadcrumbs`: Is a drop-in component to render the breadcrumbs for that page
`BreadcrumbSegment`: Is a component that we can extend that will render a breadcrumb segment.
`BreadcrumbLink`: Creates the link to page from breadcrumb.